addOns/webuipoc/webuipoc.gradle.kts

```diff
@@ -25,7 +25,7 @@ for (dir in pocsSrcDir.listFiles()!!) {
             tasks.register<NpmTask>("installPoc${normalizedPocName}Dependencies") {
                 group = pocBuildTasksGroup
                 workingDir = dir
-                args.set(arrayListOf("ci"))
+                args.set(arrayListOf("install"))
```

Comment:
It's reverting #5525.

Reply:
should I revert these changes manually?from github > edit file

Reply:
Any way should work.